packages/wp-build/src/build.mjs

```diff
@@ -994,6 +994,15 @@ async function transpilePackage( packageName ) {
 	const emotionPlugin = babel( {
 		filter: /\.[jt]sx?$/,
 		config: {
+			presets: [
```

Comment:
This is restores a behavior that we abandoned before. So I don't really think that this is the right path forward.